config.sample.yaml

```diff
@@ -27,3 +27,7 @@ serve:
   github_owner: __CHANGE_ME__
   github_app_id: __CHANGE_ME__
   github_app_installation_id: __CHANGE_ME__
+
+alert_recipients:  # Optional
+  - name: __CHANGE_ME__
+    email: __CHANGE_ME__@company.com
```

Comment:
```suggestion
    email: __CHANGE_ME__  # e.g. employee@example.com
```

Reply:
works for me!

Reply:
want me to commit that suggestion, or you?